templates/applicantForm.phtml

```diff
@@ -193,6 +193,44 @@
                 </div>
             </section>
 
+            <section class="applicationFormSection">
+                <div data-toggle="collapse" class="collapse-header" data-target="#studentInfo" aria-expanded="false" aria-controls="personalDetails">
+                    <h4>Student Info</h4>
+                    <i class="glyphicon glyphicon-chevron-down"></i>
+                </div>
+
+                <div id="studentInfo" class="collapse">
+                    <label for="github-username">Github Username</label>
+                    <div id="githubError" class="alert hidden formItem_alert"></div>
+                    <input id="github-username" name="github-username" class="submitApplicant form-control" type="text" placeholder="Github Username" value= "">
```

Comment:
Please remove the space between the equals and double quotes :) (pedantic innit)